src/API/Endpoints/Reports/Endpoint.php

```diff
@@ -59,18 +69,29 @@ public function register_route() {
 					'callback'            => array( $this, 'handle_request' ),
 					'permission_callback' => array( $this, 'permissions_check' ),
 					'args'                => array(
-						'start' => array(
+						'start'    => array(
 							'type'              => 'string',
 							'required'          => true,
 							'validate_callback' => array( $this, 'validate_date' ),
 							'sanitize_callback' => array( $this, 'sanitize_date' ),
 						),
-						'end'   => array(
+						'end'      => array(
 							'type'              => 'string',
 							'required'          => true,
 							'validate_callback' => array( $this, 'validate_date' ),
 							'sanitize_callback' => array( $this, 'sanitize_date' ),
 						),
+						'currency' => array(
+							'type'              => 'string',
+							'required'          => true,
+							'validate_callback' => array( $this, 'validate_currency' ),
+						),
+						'testMode' => array(
+							'type'              => 'string',
```

Comment:
Instead of expecting a "true" or "false" string and validating that, just set the type to `'boolean'`.

Reply:
Good thought, made the change and removed the validate_test_mode method. Ref: https://github.com/impress-org/givewp/commit/0c2188c1b9fa803b317572b855ab7ff3d0bd28b1